### DISCUSSION TITLE
proposal-seminar-thesis

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#2] [Announcements]
# https://aprd.ir/proposal-seminar-thesis/

نکاتی در مورد مسیر آکادمیک در دوران ارشد

https://aprd.ir/proposal-seminar-thesis/

**Comment by pourmand1376** [comment]
این کامنت از سایت Graphcomment - که قبلا استفاده می‌کردم - استخراج شده است. 

**نویسنده کامنت:** «محمدحسین موحد»
**متن:** سلام.
ببخشید آیا این امکان وجود دارد که یک توضیحاتی هم برای هوش مصنوعی در بازار کار بدید؟ مثلا یک نقشه راه برای دانشجویانی که می خواهند وارد بازار کار شوند چه پیشنیازهایی و چه منابعی را پیشنهاد می دهید.  
یا حوزه های دیگر برای بازار کار.

**Reply by pourmand1376** [reply]
سلام محمدحسین جان
به نظرم هنوز انقدر بلد نیستم که بتونم نقشه راه به کسی بدم ولی می‌تونم منابعی که به نظر خودم مفید بوده و مطالعه کردم رو به اشتراک بذارم. به زودی اینکار رو انجام میدم.
